5.sc_Morphology_Logistic_Regression/2.test_model/test_single_class_model.ipynb

```diff
@@ -6,69 +6,62 @@
    "id": "b551a2fa",
```

Comment:
<p>The imports look clean, good job!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>